Adding optional image location for workloads

dry923 opened the pull request:
This adds an optional workload argument image which takes the image location. This is to make debugging and testing changes easier by allowing you to pass a different image location without having to rebuild the operator image.

dry923 commented:
/rerun all

jtaleric commented:
> roles/fio-distributed/templates/.client.yaml.swp Is also added, probably not intentional. Can you also please add small documentation on how one can use it? re: update the cr with the vars?

Seems like a good opportunity to add to our gitignore.

dry923 commented:
> roles/fio-distributed/templates/.client.yaml.swp Is also added, probably not intentional. Can you also please add small documentation on how one can use it? re: update the cr with the vars?

Good call. Removed the swp file and added *.swp to gitignore. Additionally, I added a note in the README on the optional arguement.

aakarshg commented on README.md:
```diff
@@ -33,6 +33,31 @@ un-block it. The benchmarks mentioned above that state `Used` for Reconciliation
+## Optional workload images
+Optional locations for workload images can now be added easily without the need to rebuild the operator.
+To do so in the workload args section of the CR add image: [location]
+NOTE: This is not a required arguement. If omitted it will default to the currently verified workload image.
```
This doesn't work for ycsb, maybe we should add that to note as well?

dry923 replied:
Noted in the Readme
